projects.html

```diff
@@ -1,25 +1,131 @@
-                <a href="#project1">Project 1</a>
-                <a href="#project2">Project 2</a>
-                <a href="#project3">Project 3</a>
+    <body  class="body">
+
+        <div class="navbar">
+            <div style="align-items: center">
+                <img src="images/TCLBlack-bg.png" alt="GCU Tech Club Logo" class="center"
+                    style="padding-top: 5px; width: 10%" />
             </div>
-        </header>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About Us</a></li>
+                <li><a href="calendar.html">Calendar</a></li>
+                <li class="dropdown">
+                    <a href="committee.html">Committees ▼</a>
+                    <div class="dropdown-content">
+                        <a href="committee.html#cyber">Cybersecurity</a>
+                        <a href="committee.html#embedded">Embedded Development</a>
+                        <a href="committee.html#girls-who-code">Girls Who Code</a>
+                        <a href="committee.html#hardware">Hardware</a>
+                        <a href="committee.html#web-dev">Website Development</a>
+                    </div>
+                </li>
+                <li><a href="project.html">Projects</a></li>
+                <li><a href="merch.html">Merch</a></li>
+                <li><a href="contact.html">Contact Us</a></li>
+            </ul>
+        </div>
+
+
+
+
+
+        <!--My code will go here-->
+
+     <div id="rcorners1">
+        <div id="rcorners2">
+
+<div class="slideshow-container" style="left: 20%;">
+
+<div class="mySlides fade" style="max-width: 100;">
+  <div class="numbertext">1 / 3</div>
+  <div id="outer">
+    <div id="inner">
+  <img src="luffy.jpg" style="width:60%">
+    </div>
+  </div>
+
+  <div class="text"></div>project#1
+</div>
+
+<div class="mySlides fade" style="max-width: 100;">
+  <div class="numbertext" >2 / 3</div>
+  <img src="luffy vs kaido.jpg" style="width:50%">
+  <div class="text"></div>project #2
+</div>
+
+<div class="mySlides fade" style="max-width: 100;">
+  <div class="numbertext">3 / 3</div>
+  <img src="straw haats.jpg" style="width:60%">
+  <div class="text"></div>project#3
+</div>
+
+</div>
+<br>
+
+<div style="text-align:center">
+  <span class="dot"></span> 
+  <span class="dot"></span> 
+  <span class="dot"></span> 
+</div>
+
+<script>
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("dot");
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none";  
+  }
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}    
+  for (i = 0; i < dots.length; i++) {
+    dots[i].className = dots[i].className.replace(" active", "");
+  }
+  slides[slideIndex-1].style.display = "block";  
+  dots[slideIndex-1].className += " active";
```

Comment:
We can actually use DOM `classList` property here:
```suggestion
  for (const dot of dots) {
    dot.classList.remove('active');
  }
  slides[slideIndex-1].style.display = "block";  
  dots[slideIndex-1].classList.add('active');
  ```